apps/monograph/app/components/monographpost/index.tsx

```diff
@@ -90,6 +90,13 @@ function generateTableOfContents() {
         : level;
     currentHeading = headingLevel;
 
+    const isCalloutHeading =
+      !heading.previousSibling &&
+      heading.parentElement?.classList.contains("callout");
```

Comment:
Why not do `heading.closest('.callout');`?

Read more about `closest` here: https://developer.mozilla.org/en-US/docs/Web/API/Element/closest